openmicroscopy/openmicroscopy
# removing java webstart

This removes Java Webstart and moves custom index page to webclient.

To test on server:
- go to https://trout/merge/webstart/ and check if you get 404
- go to https://trout/merge/webclient/ and check if you can log in, application should work as normal
- check if insight icon is not present on the top bar

To test locally:
- set: ``bin/omero config set omero.web.index_template 'webclient/index.html'``
- start OMERO.web and go to http://localhost:4080/ and check if you are redirected to the http://localhost:4080/index/ and the page shows ``This is index template example``. Click on the webclient link. You should be able to log in as normal.

--no-rebase
